providers/base/bin/camera_test.py

```diff
@@ -193,8 +194,10 @@ def detect(self):
                 "    version: %s.%s.%s"
                 % (cp.version >> 16, (cp.version >> 8) & 0xff,
                    cp.version & 0xff))
+            capture_cability = cp.capabilities & V4L2_CAP_VIDEO_CAPTURE or \
```

Comment:
typo: I guess you meant `capability`, but maybe it's better to stick to the plural anyway (`capture_capabilities`).

Reply:
Fix the typo and append the Bugfix to the title.

I will land the unit test latter.